repo/packages/R/riak-kv/0/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "packagingVersion": "2.0",
-  "name": "riak",
-  "version": "0.1.1",
+  "name": "riak-kv",
+  "version": "1.3.0-kv-2.1.4",
   "framework": true,
-  "tags": ["database", "riak", "NoSql"],
+  "tags": ["mesosphere", "framework", "database", "riak", "riak-kv"],
```

Comment:
please remove tag `mesosphere` since this package isn't developed by mesosphere.